lib/v2/screens/send_confirmation/send_confirmation_screen.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/components/full_page_error_indicator.dart';
+import 'package:seeds/v2/components/full_page_loading_indicator.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/send_confirmation/components/transaction_details.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/send_confirmation_bloc.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/viewmodels/send_confirmation_arguments.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/viewmodels/send_confirmation_events.dart';
+import 'package:seeds/v2/screens/send_confirmation/interactor/viewmodels/send_confirmation_state.dart';
+import 'package:seeds/v2/utils/cap_utils.dart';
+
+/// SendConfirmation SCREEN
+class SendConfirmationScreen extends StatelessWidget {
+  final SendConfirmationArguments arguments;
```

Comment:
If you get the arguments in the constructor? I see that this is passed by navigation not by constructor.
```
            NavigationService.of(context).navigateTo(
                Routes.sendConfirmationScreen,
                SendConfirmationArguments(
                  account: state.pageCommand.resultData.accountName,
                  name: state.pageCommand.resultData.name,
                  data: state.pageCommand.resultData.data,
                ),
                true);
```

As far I know you can only get the navigation arguments in the build method
```
...
  @override
  Widget build(BuildContext context) {
    // TODO(raul): I do not like this way to retrive a value from navigation, https://github.com/JoinSEEDS/seeds_light_wallet/issues/500.
    _nameController.text = ModalRoute.of(context).settings.arguments;
...
```

Reply:
seems to be working like this?